composer.lock

```diff
@@ -2845,16 +2854,16 @@
         },
         {
             "name": "vimeo/psalm",
-            "version": "5.23.1",
+            "version": "5.26.1",
```

Comment:
The locked dependency still points at psalm v5, which means that we aren't running SA against v6: what happens if we only allow v6?

Reply:
@Ocramius do you know why the infection was fixed on a version and no version constraint was used :)?

Reply:
And indeed, the infection API did change exactly there :-P 

I'm locking down to the patch version again, as it should be.

Reply:
3e06be28f86dab8c23697245904009765827dff7